specification/sql/resource-manager/Microsoft.Sql/preview/2019-06-01-preview/FailoverManagedInstance.json

```diff
@@ -0,0 +1,113 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2019-06-01-preview",
+    "title": "SqlManagementClient",
+    "description": "The Azure SQL Database management API provides a RESTful set of web APIs that interact with Azure SQL Database services to manage your databases. The API enables users to create, retrieve, update, and delete databases, servers, and other entities."
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Sql/managedInstances/{managedInstanceName}/failover": {
+      "post": {
+        "tags": [
+          "ManagedInstances"
+        ],
+        "description": "Failovers a managed instance.",
+        "operationId": "ManagedInstances_Failover",
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
```

Comment:
I was getting 

> The subscription ID that identifies an Azure subscription.
, ---
The ID of the target subscription.
).
  Error: '$.parameters.SubscriptionIdParameter.description' has incompatible values (---
The subscription ID that identifies an Azure subscription.
, ---
The ID of the target subscription.
).
>
when I referenced `common-types/resource-management/v1/types.json#/parameters/SubscriptionIdParameter` and didn't know how to fix that.

Reply:
Can you try update this to common-type AND remove the subscription id definition below on Line 95? I think that should do it. In the meantime, I remove the change requested status. Thx

Reply:
I did try, but I was getting the error when execute `autorest specification\sql\resource-manager\readme.md --azure-validator=true` (I pasted error in comment above).
Can we just leave it as is? I would really appreciate to merge this changes, so I could continue with client SDKs